coss/opensource_clubs/models.py

```diff
@@ -10,7 +10,7 @@
 from wagtail.wagtailsnippets.edit_handlers import SnippetChooserPanel
 
 
-class CategoryLandingPage(Page):
+class HomePage(Page):
```

Comment:
When I am trying to add this as the landing page of the project (local env), I am getting the following error:

`AttributeError at /
'str' object has no attribute 'relative_url' `

Reply:
I updated the PR to add a check for the case that the Category Landing Page is not created yet.